addons/ingress-nginx/addon.rb

```diff
@@ -15,6 +15,7 @@
     optional(:node_selector).filled(:hash?)
     optional(:default_backend).schema {
       optional(:image).filled(:str?)
+      optional(:secret_name).filled(:str?)
```

Comment:
IMO the name could be something bit more descriptive. In this case the secret is being used as `imagePullSecret` hence I'd prefer the name to also reflect it's usage.